app/build.gradle

```diff
@@ -14,15 +14,6 @@ android {
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
-    testOptions {
```

Comment:
@sagarprasad63574 Could you please run all tests, make sure they pass? I think we need these testOptions for some of them.

Reply:
ok